cNVMe/Main.cpp

```diff
@@ -23,19 +23,40 @@
 Main.cpp - An implementation file for the Main entry
 */
 
+#include "Driver.h"
 #include "Strings.h"
 #include "Tests.h"
 
+
 #include <iostream>
 
 using namespace cnvme;
 using namespace cnvme::command;
+using namespace cnvme::driver;
 
 int main()
 {
 	// This is testing code.
 	LOG_SET_LEVEL(2);
 
+	Driver driver;
+	UINT_32 BUF_SIZE = 8192;
+	BYTE* buffer = new BYTE[BUF_SIZE];
+	memset(buffer, 0, BUF_SIZE);
+
+	DRIVER_COMMAND* d = (PDRIVER_COMMAND)buffer;
+	d->Timeout = 5;
+	d->Command.DWord0Breakdown.OPC = 6;
+	d->Command.DWord10 = 1;
+	d->TransferDataDirection = READ;
+	d->TransferDataSize = 4096;
+
+	driver.sendCommand((UINT_8*)d, BUF_SIZE);
+
+	char c = '1';
+	driver.sendCommand((UINT_8*)d, BUF_SIZE);
```

Comment:
"char c"? Where is that getting used? Why are we sending the command again?

Reply:
It was my way of having a place to put a breakpoint when manually testing. Will cleanup before merge.